docs/doxygen/migration/shadow.txt

```diff
@@ -0,0 +1,62 @@
+/**
+@page shadow_migration Shadow
+@brief How to migrate an Shadow application from v3 to 202009.00.
+
+The Device Shadow library has been redesigned in the Long Term Support (LTS) release.  The new design has the following differences than the Version 3 of the AWS IoT Device SDK for Embedded C.
+
+- The new Device Shadow library is completely decoupled from an underlying MQTT library.  The Version 3 library depended on the same version of MQTT library in the AWS IoT Device SDK for Embedded C. <br>
```

Comment:
```suggestion
- The new Device Shadow library is completely decoupled from an underlying MQTT library.  The Version 3 library depends on the same version of the MQTT library in the AWS IoT Device SDK for Embedded C. <br>
```

Reply:
Also can you change all Version 3 to "v3"?

Reply:
I also think the first sentence is enough. will remove the one about v3.